doc/how_to_guides/isaac_panda/.docker/Dockerfile

```diff
@@ -13,14 +13,19 @@ RUN apt-get update && apt-get upgrade -y && rosdep update
 
 # Install packages required to run the demo
 RUN apt-get install -y --no-install-recommends \
-    ros-humble-moveit \
-    ros-humble-moveit-resources
+    ros-humble-moveit
 
 # Create Colcon workspace and clone the needed source code to run the demo
 RUN mkdir -p /root/isaac_moveit_tutorial_ws/src
 WORKDIR /root/isaac_moveit_tutorial_ws/src
 RUN git clone https://github.com/PickNikRobotics/topic_based_ros2_control.git
+
+WORKDIR /root/isaac_moveit_tutorial_ws/src
+RUN git clone https://github.com/ros-planning/moveit_resources.git -b humble
+
+WORKDIR /root/isaac_moveit_tutorial_ws/src
```

Comment:
What's wrong with the binary version it was installing? The rename of `topic_based_ros2_control` was included in the initial release of the tutorial.

Reply:
@tylerjw when you switched humble to build main we lost the commit that included the hardware_interface name change. I made a [PR here](https://github.com/ros-planning/moveit_resources/pull/172) to correct the issue.

Reply:
The problem is we are using the `humble` branch of the tutorials and not `main` but we are using the `main` version of MoveIt. We should move this tutorial onto the `main` version of the tutorials.